src/main/java/board/GameMainView.java

```diff
@@ -3,28 +3,81 @@
 import javax.swing.*;
 import javax.swing.border.EmptyBorder;
 import java.awt.*;
+import java.awt.event.*;
 
 public class GameMainView extends JFrame {
-
+    
     private static int rows = 4;
     private static int cols = 4;
     private static board.BoardView boardView = new board.BoardView(rows, cols);
-    private final JPanel panel = new JPanel(new BorderLayout());
-
-    public GameMainView(){
+    private static final JPanel panel = new JPanel(new BorderLayout());
+    
+    private static final JMenuBar mnubarMain = new JMenuBar();
+    
+    private static final JMenu mnuDifficulty = new JMenu("Difficulty");
+    private static final String[] difficulty = {"Easy", "Medium", "Hard"};
+    private static final JMenuItem mnuitmDifficulty[] = new JMenuItem[difficulty.length];
+    
+    public GameMainView(){       
         setTitle("MEMORY GAME");
         setSize(900, 750);
         setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
         setLocation(340, 80);
-        panel.setBorder(new EmptyBorder(20, 20, 20, 5));
+        setLocationRelativeTo(null);
 
+        panel.setBorder(new EmptyBorder(20, 20, 20, 5));
         panel.add(boardView);
         add(panel, BorderLayout.CENTER);
-
+                
+        for (int i = 0; i < mnuitmDifficulty.length; i++){
+            mnuitmDifficulty[i] = new JMenuItem(difficulty[i]);
+            mnuitmDifficulty[i].addActionListener(new DifficultyAction(i));
+            mnuDifficulty.add(mnuitmDifficulty[i]);
+        }
+       
+        mnubarMain.add(mnuDifficulty);
+        
+        setJMenuBar(mnubarMain);
+    
     }
 
-    public static void main(String[] args) {
+    public static void main(String[] args) {      
         new GameMainView().setVisible(true);
+       
+    }
+    
+    private static class DifficultyAction implements ActionListener{
+        int item = 0;
+        
+        public DifficultyAction(int item) {
+            this.item = item;
+            
+        }
+
+        @Override
+        public void actionPerformed(ActionEvent e) {
```

Comment:
get rid of the switch statement. you don't need it if you are passing in the size.